templates/cli/lib/commands/generic.js.twig

```diff
@@ -122,8 +151,51 @@ const logout = new Command("logout")
             sdk: client
         })
 
-        globalConfig.setCookie("");
-        success()
+        globalConfig.removeLogin(accountId);
+    } catch (e) {
+        error('Unable to log out, removing locally saved session information')
+    }
+    globalConfig.removeLogin(accountId);
+}
+
+const logout = new Command("logout")
+    .description(commandDescriptions['logout'])
+    .configureHelp({
+        helpWidth: process.stdout.columns || 80
+    })
+    .action(actionRunner(async () => {
+        const logins = globalConfig.getLogins();
+        const current = globalConfig.getCurrentLogin();
+
+        if (current === '') {
+            return;
+        }
+        if (logins.length === 1) {
+            await singleLogout(current);
+            success();
+
+            return;
+        }
+
+        const answers = await inquirer.prompt(questionsLogout);
+
+        if (answers.accounts) {
+            for (let accountId of answers.accounts) {
+                globalConfig.setCurrentLogin(accountId);
+                await singleLogout(accountId);
+            }
+        }
+
+        const leftLogins = globalConfig.getLogins();
```

Comment:
shouldn't this method be called getSessions() ?

Reply:
This is to set the next account in line as the `current` one